### ISSUE TITLE
PDF is auto-downloaded on page load

### ISSUE THREAD

**Issue by LeskoIam** [karakeep-app/karakeep#711]
### Describe the Bug

If PDF is uploaded to hoarder it is auto-downloaded on every page refresh.

### Steps to Reproduce

1. Uploaded PDF document to hoarder (drag&drop)
2. Refresh page.
3. Previously uploaded document is downloaded (without clicking anything. except refresh button in browser)

### Expected Behaviour

No file should be downloaded.

### Device Details

133.0 (64-bit) Windows 10 Pro (22H2)

### Exact Hoarder Version

v0.19.0

**Comment by MohamedBassem** [comment]
Yeah. That's an unfortunate side effect from how we render the bookmark cards. We should definitely fix that.

**Comment by MohamedBassem** [comment]
Fixed by #995
